utils/patches/computeAngle/src/test_cart.cpp

```diff
@@ -0,0 +1,45 @@
+#include "cart.h"
+
+#include <iostream>
+
+#include <cmath>
+
+int count = 1;
+
+// callback every 100ms
+class DisplayAngleCallback: public Cart::CompAngleCallback {
+  public: int a = -87;
+  int b = -51;
+  int c = -91;
+  float angle_updated;
+
```

Comment:
static const int offset_a = 52; 

use the variable offset_a  here instead of a number, or declare a new one as offset.

This ensure strong type definations as suggested by professor

Reply:
FIxed.